templates/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
```

Comment:
Kindly replace the `nav` tag with the `header` tag for semantic reasons 🤗
- Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.